app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,68 @@
+class TasksController < ApplicationController
+  # RENAME FOLDER TASKS & change folder/controller associations
+  before_action :find_task, only: [:show, :edit, :update, :complete, :delete]
+
+  def index
+    @tasks = Task.all
+    @all_the_people = Person.all
+  end
+
+  def show
+    @all_the_people = Person.all
+  end
+
+  def new
+    @task = Task.new
+    @all_the_people = Person.all
+    @url = "/tasks/new"
+  end
+
+  def create
+    @task = Task.create(create_params)
+    redirect_to "/"
+  end
+
+  def complete
+      @task.completed = "Yes"
+      @task.date = Time.now
+    @task.save
+    redirect_to "/"
+  end
+
+  def edit
+    @all_the_people = Person.all
+    @url = "/tasks/#{@task.id}/edit"
+  end
+
+  def update
+    # @task.id => return id associated w/task
+    @task.name = create_params[:name]
```

Comment:
You should be able to use the update method to update all of these fields rather than specifying them individually